python/lsst/obs/base/instrument.py

```diff
@@ -24,8 +24,19 @@
 
 import os.path
 from abc import ABCMeta, abstractmethod
+import astropy.time
 
-from lsst.daf.butler import TIMESPAN_MIN, TIMESPAN_MAX
+from lsst.daf.butler import TIMESPAN_MIN, TIMESPAN_MAX, DatasetType, DataCoordinate
+from lsst.utils import getPackageDir
+
+# To be a standard text curated calibration means that we use a
+# standard definition for the corresponding DatasetType.
+StandardCuratedCalibrationDatasetTypes = {
```

Comment:
Looking at this again I don't think this should be capitalized.

Reply:
Could probably make an argument for it to be `STANDARD_CURATED_CALIBRATION_DATASET_TYPES`

Reply:
@ktlim I'm not exactly sure what naming scheme should be used. It's a file global but I'm not exporting it.